en/gps_compass/rtk_gps_locosys_r2.md

```diff
@@ -0,0 +1,109 @@
+# LOCOSYS Hawk R2 RTK-Compass GPS
+
+The [LOCOSYS Hawk R2](https://www.locosystech.com/en/product/hawk-r2.html) is a dual-frequency [RTK GPS module](../gps_compass/rtk_gps.md) receiver designed for compatibility with Pixhawk.
+Additionally, it is equipped with an e-compass.
+The only difference between this module and the [LOCOSYS Hawk R1](rtk_gps_locosys_r1.md) is the presence of an E-compass on Hawk R2.
+
+The module can act as rover when installed on the aircraft.
+The receiver is capable of concurrently tracking all global civil navigation systems, including GPS, GLONASS, GALILEO, BEIDOU and QZSS.
+It acquires both L1 and L5 signals at the same time while providing the centimeter-level RTK positioning accuracy.
+
+The built-in lightweight helical antenna enhances RTK positioning stability.
+The fast time-to-first-fix, RTK convergence, superior sensitivity, low power consumption make it a better choice for Pixhawk-based platform UAVs.
+
+Main features include:
+- Centimeter-level RTK high position accuracy+ integrated 3-axis e-compass
+- Concurrent reception of L1 and L5 band signals
+- Support GPS, GLONASS, BEIDOU, GALILEO, QZSS
+- Capable of SBAS (WAAS, EGNOS, MSAS, GAGAN)
+- Support 135-channel GNSS
+- Fast TTFF at low signal level
+- Free hybrid ephemeris prediction to achieve faster cold start
+- Default 5Hz, up to 10 Hz update rate (SBAS support 5Hz only)
+- Build-in super capacitor to reserve system data for rapid satellite acquisition
+- Build-in 3 axis compass function
+- Three LED indicator for Power, PPS and Data transmit
+
+:::note
+This module can be used with PX4 v1.13 or above.
+:::
+
+![LOCOSYS Hawk R2](../../assets/hardware/gps/locosys_hawk_a1/locosys_hawk_a1_gps.png)
+
+
+## Where to Buy
+
+* [LOCOSYS Hawk R2](https://www.locosystech.com/en/product/hawk-r2.html)
+
+## Kit Contents
+
+An RTK GPS kit includes:
+- 1x GPS Module
+- 1x Helix antenna
+- 1x 6-pin JST-GH 
+
+
+## Configuration
+
+RTK setup and use on PX4 via *QGroundControl* is largely plug and play (see [RTK GPS](../advanced_features/rtk-gps.md) for more information).

```

Comment:
This had some spurious information and was still on R1 (until I subedited). Is it correct? Is there any special setup for the compass?

Reply:
I do not know how to put this actually. The idea is this is a Rover module, so it has to come on the Autopilot and in compass calibration be used. Although this compass is not that much satisfactory and internal FCU compass is better to be used unless it is mounted away a bit from the vehicle core electronics and power.

Reply:
@farhangnaderi If it is worse than the FC compass then it is truly awful - that's sitting right in the electronics. However I don't think that matters - we always say "mount the compass away from the electronics"  - the compass detection will work out which one it wants to use. Or are you saying this is so bad they really should not have made this FC variant?

YOu say below this comes with a 6 pin JST connector for the GPS. I presume that the I2C pins 7, 8 are for the compass and no cable is supplied for those pins? 
In that case we should have a note about the compass that says something like:

> No cable is provided for the compass. If needed, the compass part can be connected via the I2C pins 7,8 using a custom cable, and configuration should be the same as for [any other compass](http://docs.px4.io/main/en/gps_compass/#compass-configuration).

If you really think it is horrible, we could further add something along the lines that the compass part is particularly succesptible to electromagnetic noise, and is not recommended on smaller frames?

Reply:
@hamishwillee considering the fact that compass has to be away from the FCU, I guess it is fine to have it kept like this. In my use case I brought it near to FCU and calibrated which led to PX4 choosing the other external compass having less noisy data. 

The cable comes with it already and is capable of connecting to Pixhawk directly. I just do not know how to name it. The cable looks like this both sides:
![Image](https://user-images.githubusercontent.com/46557204/200947512-8c507113-6b5e-46f2-99b2-1494c2af3b64.jpeg)

Reply:
Thats great. Can see from that the cable does both compass and GPS. I'll do a last check over and merge.